packages/ui/src/helpers/authenticator/translations.ts

```diff
@@ -0,0 +1,139 @@
+import {
```

Comment:
Maybe we should rename this file from `translations` to `authenticatorTextUtil`?

Reply:
Good point, will do.